airbyte-webapp/src/components/connection/CatalogTree/next/CatalogTreeTableRow.tsx

```diff
@@ -66,18 +66,17 @@ export const CatalogTreeTableRow: React.FC<StreamHeaderProps> = ({
       <CatalogTreeTableCell size="small">
         <Switch size="sm" checked={stream.config?.selected} onChange={onSelectStream} disabled={disabled} />
       </CatalogTreeTableCell>
-      {/* <Cell>{fieldCount}</Cell> */}
-      <CatalogTreeTableCell>
+      <CatalogTreeTableCell withTooltip>
         <Text size="md" className={styles.cellText}>
           {stream.stream?.namespace || <FormattedMessage id="form.noNamespace" />}
         </Text>
       </CatalogTreeTableCell>
-      <CatalogTreeTableCell>
+      <CatalogTreeTableCell withTooltip>
         <Text size="md" className={styles.cellText}>
           {stream.stream?.name}
         </Text>
       </CatalogTreeTableCell>
-      <CatalogTreeTableCell size="large">
+      <CatalogTreeTableCell size="large" withTooltip={disabled}>
```

Comment:
This is one of the main reasons we're introducing the Tooltip -- the sync mode "incremental | deduped + history" always has overflow.  We want a Tooltip on it.

Reply:
So we want the value of the dropdown as a tooltip here?

Reply:
Yeah!  That was the idea as I understood it.